.eslintrc.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
`.eslintrc` ist die Konfiguration von `eslint`. Analog gibt es zu den anderen Tools auch solche Konfigurationsdateien.

Hier definieren wir erstmal welche Regelwerke wir erweitern, in diesem Fall sind das:

- react-app: Standard-Regelwerk von create-react-app. Das ist relativ lasch und zeigt nur Fehler auf, aber keine Styling-Guidelines
- airbnb: Ist eines der größten Standard-Regelwerke wo auch Sachen wie Einrückung, Methodendefinition, Strichpunkte, Anführungszeichen usw. definiert werden
- jest/all: Regelwerk für unsere Tests (jest ist unser Testing-Framework), damit wir uns da an gewisse Standards halten (kann ich euch bei Bedarf im Detail erklären)
- jsx-a11y: Das ist ein Sonderregelwerk, das aber auch recht beliebt ist. Hier werden Regeln für a11y im JSX-Kontext (also React) definiert, also z.B. dass man onClick Handler auch nur auf klickbare Elemente definiert usw. Sonst kann es theoretisch passieren, dass die App in manchen Browsern nicht nutzbar ist. Falls ihr a11y noch nicht kennt: https://a11yproject.com/